packages/playground/src/views/vms_view.vue

```diff
@@ -26,6 +26,7 @@ export default {
           "Deploy a full virtual machine on Threefold Grid. Full VM allows you to have a complete image with a custom kernel optimized for your own usecase.",
         icon: "vm.png",
         route: DashboardRoutes.VirtualMachines.FullVirtualMachine,
+        // releaseDate: { year: 2024, month: 9, day: 4 },
```

Comment:
Is that happen by mistake? the commented code

Reply:
No, it's the format to remind who will edit how it should be like

Reply:
I think should be added in docs or, in doc string

Reply:
I think @Mahmoud-Emad is right, I should remove ```releaseDate``` comment it's interface job so there is no use for the doc string too

Reply:
done